### DISCUSSION TITLE
What does it really mean to give an interactive CSR?

### DISCUSSION THREAD

**Discussion by epijim** [rinpharma/positconf-roundtables-2023#17] [General]
Early feedback from a regulator has been concern about a shiny CSR, which brings up interesting points about allowing near limitless exploration of a dataset carefully created and powered for a specific hypothesis. 

Tagging @mikesmith for bringing this general topic up.

What are our recommendations on releasing shiny apps in a responsible manner, with a key use case being regulatory use but including the sharing of apps with non-technical users (e.g affiliates, access users, etc)?

**Comment by harveyl888** [comment]
I know this is not the place to discuss (more to propose) but ...
I've been working on a question-based-approach for statisticians to communicate results to clinicians but always thought it could be applied to a CSR.  Basic concept is that the end-user does not have limitless exploration - what you see is a curated set of questions, each of which leads to one or more outputs.  The framework I showed at last year's R/Pharma (and Tiina presented at UseR last month) takes parameterized input and generates the outputs in realtime as questions are asked.  I have lots more to say about this but not in this forum.
